server/services/reportsService.ts

```diff
@@ -202,11 +235,10 @@ export default class ReportsService {
     const prisoners = await this.getPrisonersForPrisonCloseToHdced(username, prisonId, bookingIds)
     const offenderNumbers = prisoners.map((p) => p.prisonerNumber)
     const probationDetails = await this.getProbationDetails(username, offenderNumbers)
-    const unallocatedProbationDetails = probationDetails.filter((pd) => ReportsService.isUnallocated(pd))
```

Comment:
removed this filter as now pass all probation details to decorate licences function so we do not discard those prisoners who have no probation details and are therefore unallocated to a com from the com unallocated report.